src/App/index.js

```diff
@@ -40,14 +40,12 @@ const Components = React.lazy(() => import(/* webpackChunkName: "components.chun
 
 const Identity = React.lazy(() => import(/* webpackChunkName: "core.chunk" */ "./Identity/index.js"));
 
-const Examples = React.lazy(() => import(/* webpackChunkName: "examples.chunk" */ "./Examples/index.js"));
-
-const Utilities = React.lazy(() => import(/* webpackChunkName: "utilities.chunk" */ "./Utilities/index.js"));
-
-const Patterns = React.lazy(() => import(/* webpackChunkName: "utilities.chunk" */ "./Patterns/index.js"));
+const Patterns = React.lazy(() => import(/* webpackChunkName: "patterns.chunk" */ "./Patterns/index.js"));
```

Comment:
curiosity question: what are those " `/* webpackChunkName: .... */`  " doing ?
should we remove them? what is (or was) the point of having them?

Reply:
I have no idea

Reply:
based on the webpack docs: https://webpack.js.org/api/module-methods/#magic-comments 
those comments are actually taken into account ("magic comments")
therefore we should NOT remove them, they are used by webpack even though they are comments.
still no clue what "chunks" are in webpack though